source/binding/Statements.cpp

```diff
@@ -757,6 +784,12 @@ bool ConditionalStatement::verifyConstantImpl(EvalContext& context) const {
     return true;
 }
 
+void ConditionalStatement::serializeTo(ASTSerializer& serializer) const {
+    serializer.write("cond", cond);
+    serializer.write("ifTrue", ifTrue);
+    serializer.write("ifFalse", ifFalse);
```

Comment:
Hmm, I'm a little surprised this compiles, ifFalse is a pointer.

Also you'll have to check ifFalse for null. The general practice throughout the library is that if it's a pointer that's not in a list and not wrapped in `not_null<>`, it can potentially be null.